app/models/course.rb

```diff
@@ -21,6 +21,10 @@ class Course < ApplicationRecord
     npq-senco
   ].freeze
 
+  def schedule_for(cohort:)
```

Comment:
discussed in person: think about defaults at this level and required attributes in the course group class